corral/bundle/bundle.pony

```diff
@@ -144,7 +147,8 @@ class Bundle
     let tran_deps = transitive_deps()
     let roots = recover trn Array[String] end
     for dep in tran_deps.values() do
-      let dr = Path.join(corral_dir(), Path.join(dep.flat_name(), dep.locator.bundle_path))
+      let dr = Path.join(corral_dir(),
```

Comment:
See the style guide for function argument positioning: https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#function-calls